packages/touchpoint-ui/index.html

```diff
@@ -89,20 +89,96 @@ <h1>Loreum ipsum dolor sit amet</h1>
       </p>
     </main>
     <script type="module">
-      import { create } from "/src/index.tsx";
-      const touchpoint = await create({
-        config: {
-          applicationUrl:
-            "https://bots.dev.studio.nlx.ai/c/kQwAgKCx86vJ4r0PUSxYR/ON4zgwWnVofsz2NaiD38Z",
-          headers: {
-            "nlx-api-key": "XDE2Th70VhBSlhz2YA",
-          },
-          userId: "1234-5678",
-          languageCode: "en-US",
+      import { create, html, Icons } from "/src/index.tsx";
+
+      // Example custom modality component for voice
+      const ExampleCard = ({ data, conversationHandler }) => {
+        return html`
+          <div className="bg-accent rounded-inner p-4 text-white shadow-lg">
+            <div className="flex items-center gap-2 mb-2">
+              <${Icons.Touchpoint} className="w-5 h-5" />
```

Comment:
I thought the `html` helper included the components by default? Can't we just do:

```suggestion
              <Icons.Touchpoint className="w-5 h-5" />
```

Reply:
Actually per definition it would just be `<Touchpoint`, but maybe I made a mistake there? https://github.com/nlxai/sdk/blob/08064a60b11f139c5ef74152140deda1e5b8ce00/packages/touchpoint-ui/src/index.tsx#L58

Reply:
I'm going to leave it like this for now so at least if we change it typescript catches it.